pkgs/development/python-modules/quamash/default.nix

```diff
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchPypi, isPy3k, pytest, pyqt5, pyqt ? pyqt5 }:
+buildPythonPackage rec {
+  pname = "Quamash";
+  version = "0.6.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0p0vysc1pm2vcla1g8d7wkzcsb68crsvi0m0clg5j83wpv4sccba";
+  };
+
+  propagatedBuildInputs = [ pyqt ];
+
+  disabled = !isPy3k;
+
+  # Tests trying to run windows tests
+  doCheck = false;
```

Comment:
I think we can skip that particular test in the pytest invocation.

Care to paste the output of this?

Reply:
```
running install tests
running test
running egg_info
writing Quamash.egg-info/PKG-INFO
writing dependency_links to Quamash.egg-info/dependency_links.txt
writing requirements to Quamash.egg-info/requires.txt
writing top-level names to Quamash.egg-info/top_level.txt
reading manifest file 'Quamash.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'Quamash.egg-info/SOURCES.txt'
running build_ext
Traceback (most recent call last):
  File "nix_run_setup", line 8, in <module>
    exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
  File "setup.py", line 40, in <module>
    'test': ['pytest'],
  File "/nix/store/nj4ca7icyaq28i67g5sy8kwg81kxn8jv-python3.6-bootstrapped-pip-18.1/lib/python3.6/site-packages/setuptools/__init__.py", line 143, in setup
    return distutils.core.setup(**attrs)
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/distutils/dist.py", line 955, in run_commands
    self.run_command(cmd)
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/distutils/dist.py", line 974, in run_command
    cmd_obj.run()
  File "/nix/store/nj4ca7icyaq28i67g5sy8kwg81kxn8jv-python3.6-bootstrapped-pip-18.1/lib/python3.6/site-packages/setuptools/command/test.py", line 228, in run
    self.run_tests()
  File "/nix/store/nj4ca7icyaq28i67g5sy8kwg81kxn8jv-python3.6-bootstrapped-pip-18.1/lib/python3.6/site-packages/setuptools/command/test.py", line 250, in run_tests
    exit=False,
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/main.py", line 94, in __init__
    self.parseArgs(argv)
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/main.py", line 118, in parseArgs
    self._do_discovery(argv[2:])
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/main.py", line 229, in _do_discovery
    self.test = loader.discover(self.start, self.pattern, self.top)
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/loader.py", line 341, in discover
    tests = list(self._find_tests(start_dir, pattern))
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/loader.py", line 398, in _find_tests
    full_path, pattern, namespace)
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/loader.py", line 475, in _find_test_path
    tests = self.loadTestsFromModule(package, pattern=pattern)
  File "/nix/store/nj4ca7icyaq28i67g5sy8kwg81kxn8jv-python3.6-bootstrapped-pip-18.1/lib/python3.6/site-packages/setuptools/command/test.py", line 54, in loadTestsFromModule
    tests.append(self.loadTestsFromName(submodule))
  File "/nix/store/qbyckcddc74s2fi060n8xqcn8gygrhjl-python3-3.6.8/lib/python3.6/unittest/loader.py", line 153, in loadTestsFromName
    module = __import__(module_name)
  File "/build/Quamash-0.6.1/quamash/_windows.py", line 25, in <module>
    class _ProactorEventLoop(asyncio.ProactorEventLoop):
AttributeError: module 'asyncio' has no attribute 'ProactorEventLoop'
builder for '/nix/store/lnc13gnm46dbp9v06bjrw70np5yw280i-python3.6-quamash-0.6.1.drv' failed with exit code 1
error: build of '/nix/store/lnc13gnm46dbp9v06bjrw70np5yw280i-python3.6-quamash-0.6.1.drv' failed
```

The docs says it's windows only https://docs.python.org/3/library/asyncio-eventloops.html#asyncio.ProactorEventLoop

I tried overriding the `checkPhase` (as suggested in the common issues [here](https://nixos.org/nixpkgs/manual/#automatic-tests)) but I couldn't get it to work, no tests would be run and it didn't seem like there were any tests there at that point.